.github/deploy/production.hcl

```diff
@@ -25,6 +25,9 @@ job "blockbot" {
         data        = <<EOF
 TOKEN={{ key "blockbot/discord/token" }}
 DEBUG=false
+LDAP_USERNAME={{ key "blockbot/ldap/username" }}
+LDAP_PASSWORD={{ key "blockbot/ldap/password" }}
+UID_MAP={{ key "blockbot/ldap/uid_map" }}
```

Comment:
if the uid_map is discord uid's, I think then it should be under `blockblot/discord/uid_map` rather than under ldap cause that might cause confusion in the future with future admins/webgroup members